src/main/java/javax/resource/ResourceException.java

```diff
@@ -29,7 +29,7 @@
  *   <LI> reference to another exception. Often a resource exception
  *        will be result of a lower level problem. If appropriate, this
  *        lower level exception can be linked to the ResourceException.
- *        Note, this has been deprecated in favor of J2SE release 1.4 exception
+ *        Note, this has been deprecated in favor of Java&trade; SE release 1.4 exception
```

Comment:
In the release 1.4 time frame it was not called "Java SE", it was called "J2SE", so all these updates of this sort are wrong.

Reply:
All references changed back to original J2SE